script/repl

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Usage: script/repl [ARGS...]
+# Run a repl session capable of loading all of the packages and their individual components. Any passed arguments, e.g. module names or flags, will be passed to ghci.
+
+set -e
+
+cd $(dirname "$0")/..
+
+repl_builddir=dist-repl
+
+if [[ ! -d $repl_builddir ]]; then
+  echo "$repl_builddir does not exist, first run 'cabal repl --builddir=$replbuilddir', exit, and then re-run $0"
```

Comment:
Can we do this automatically? I guess not, since the repl is an interactive process…

Reply:
Also, you made a typo here: it should be `'cabal repl --builddir=$repl_builddir’`.

Reply:
Oops, good catch!